### PR TITLE
Web auth endpoint up on heroku, communicating with blockmaster node

### PR DESCRIPTION
**Pull request by nicholeous**
- refactor interbit deploy for use outside of cli with options
- configure platform nodes to allow web auth node to communicate
- generate a new manifest for the new ACL
- add a connect option to deploy to bypass configuration if the network is already running
- tidy up for heroku 
- refactor auth endpoint to separate node and authserver and network configuration

### CODE REVIEW

File: packages/web-auth-endpoint/src/authServer.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  const githubChainAlias = 'accountsGithubAuth'
+  const githubChainId = manifest.chains[githubChainAlias]
+  const githubChain = cli.getChain(githubChainId)
+
+  app.get('/oauth/github', (request, response) => {
+    const host = request.get('host')
+    console.log('/oauth/github request from: ', host)
+    if (whitelist.indexOf(host) === -1) {
+      console.log('no host found using whitelist: ', whitelist)
+      return response.status(403).send('Nope!')
```
Comment (recurrence) [inline packages/web-auth-endpoint/src/authServer.js]: 'Nope!' ?

File: packages/web-auth-endpoint/src/networkConfig.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const DEFAULT_PORT = 8888
+
+const PORT = process.env.PORT || DEFAULT_PORT
+
+module.exports = {
+  PORT,
+  AUTH_PORT: PORT,
+  NODE_PORT: PORT + 1 > 65535 ? 5 : PORT + 1,
```
Comment (recurrence) [inline packages/web-auth-endpoint/src/networkConfig.js]: huh?

File: packages/web-auth-endpoint/src/authServer.js (hunk 1 above)
Comment (nicholeous) [reply packages/web-auth-endpoint/src/authServer.js]: Yes :)

Comment (recurrence) [reply packages/web-auth-endpoint/src/authServer.js]: How bout just a blank 403 with sendStatus then?

**Pull request merged**